adventure/api.py

```diff
@@ -63,5 +63,14 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    player_id = player.id
+    player_uuid = player.uuid
+    data = json.loads(request.body)
+    message = data['message']
```

Comment:
It's small thing, but have you thought about how you'd handle the case in which the user sends a blank message, or a message with malicious code?